common/web/gesture-recognizer/src/incomplete.ts

```diff
@@ -0,0 +1,45 @@
+namespace com.keyman.osk {
+  /**
+   * Supported events:
+   *
+   * - 'update':  a new sample (coord + timestamp) has been observed
+   * - 'cancel':  all gesture recognition on the sequence is to be cancelled
+   *                   and left incomplete.
+   * - 'end':     all gesture recognition on the sequence is to be resolved.
+   */
+  export class Incomplete<Type, UpdateType> extends EventEmitter {
```

Comment:
An incomplete what? Could this be `IncompleteGesture`? An `IncompleteSequence`? I really don't know... 

I think, either rename, or change the namespace to clarify.